api/conf/schema.json

```diff
@@ -54,6 +54,9 @@
 		"global_rule": {
 			"additionalProperties": false,
 			"properties": {
+				"create_time": {
```

Comment:
do we sync this schema.json from APISIX project?

Reply:
ping @imjoey

Reply:
> do we sync this schema.json from APISIX project?

@membphis yep, synced from APISIX project, but only the two added properties were kept in this PR, so as not to introduce unexpected problems.